lib/pages_desktop_specific/my_stack_view/wallet_view/sub_widgets/more_features/more_features_dialog.dart

```diff
@@ -61,6 +69,13 @@ class _MoreFeaturesDialogState extends ConsumerState<MoreFeaturesDialog> {
       ),
     );
 
+    // Parse otherDataJsonString to get the enableLelantusScanning value.
+    if (wallet.info.otherDataJsonString != null) {
+      final otherDataJson = json.decode(wallet.info.otherDataJsonString!);
+      enableLelantusScanning =
+          otherDataJson["enableLelantusScanning"] as bool? ?? false;
```

Comment:
Use `WalletInfoKeys.enableLelantusScanning` in place of the string key. Even if the same now, a change to only one could introduce a hard to trace bug.

Reply:
Also applied anywhere else I used "enableLelantusScanning" vs WalletInfoKeys.enableLelantusScanning